meta-luneos/recipes-luneos/nyx-modules-hybris/nyx-modules-hybris.bb

```diff
@@ -14,7 +14,7 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 
 # Let us fetch the machine-specific CMake configuration used by nyx-modules, to
 # define it only once
-FILESEXTRAPATHS:prepend := "${THISDIR}/nyx-modules:"
+FILESEXTRAPATHS:prepend := "${THISDIR}/../../recipes-webos-ose/nyx-modules/nyx-modules:"
```

Comment:
is that path correct ?

Reply:
Yes it builds like that... We need to go 2 levels up. Without the last nyx-modules it would fail.